docs/manual-CN/deploy-cluster.md

```diff
@@ -0,0 +1,146 @@
+
+示例命令如下：
+
+```
+scripts/services.sh <start|stop|restart|status|kill>
+```
+
+metas， storages 和 graphs 包含其自身的 hosts。
+
+***
+
+#### 配置引用
+
+**Meta Service** 支持如下配置属性：
+
+属性名               | 默认值            | 说明
+--------------------------- | ------------------------ | -----------
+`port`                      | 45500                    | Meta daemon 监听端口
+`reuse_port`                | true                     | 开启 SO_REUSEPORT 选项
+`data_path`                 | ""                       | 根数据目录
+`meta_server_addrs`                     | ""                       | 一系列由逗号分隔的IP地址，IP数与副本数相等，如果为空，则表明副本数为1
+`local_ip`                  | ""                       | 指定本地IP  NetworkUtils::getLocalIP.
+`num_io_threads`            | 16                       | IO 线程数
+`meta_http_thread_num`      | 3                        | meta daemon 的 http 线程数
+`num_worker_threads`        | 32                       | worker 数
+`part_man_type`             | memory                   | memory，meta
+`pid_file`                  | "pids/nebula-metad.pid"  | 存储进程 ID 的文件
+`daemonize`                 | true                     | 作为 daemon 进程运行
+`cluster_id`                | 0                        | 每个集群都惟一的 ID
+`putTryNum`                 | 10                       | 尝试生成集群 ID 的次数
+`load_config_interval_secs` | 2 * 60                   | 加载配置间隔
+`meta_ingest_thread_num`    | 3.                       | Meta daemon的 ingest 线程数
+
+**Storage Service** 支持如下配置属性：
+
+属性名                       | 默认值              | 说明
+----------------------------------- | -------------------------- | -----------
+`port`                              | 44500                      | Storage daemon 监听端口
+`reuse_port`                        | true                       | 开启 SO_REUSEPORT 选项
+`data_path`                         | ""                         | 根数据目录，多条路径由逗号分隔，对 rocksdb 引擎，一个路径为一个实例
+`local_ip`                          | ""                         |  IP 地址和监听端口共同用于标识此服务器
+`daemonize`                         | true                       | 作为 daemon 进程运行
+`pid_file`                          | "pids/nebula-storaged.pid" | 存储进程 ID 的文件
+`meta_server_addrs`                 | ""                         | meta server 地址列表，格式为 ip1:port1, ip2:port2, ip3:port3
+`store_type`                        | "nebula"                   | storage daemon 使用的 KVStore 类型，可选类型为 \"nebula\"和\"hbase\"
+`num_io_threads`                    | 16                         | IO 线程数
+`storage_http_thread_num`           | 3                          | storage daemon 的 http 线程数
+`num_worker_threads`                | 32                         | worker 数
+`engine_type`                       | rocksdb                    | rocksdb, memory...
+`custom_filter_interval_secs`       | 24 * 3600                  | 触发自定义压缩间隔
+`num_workers`                       | 4                          | worker 线程数
+`rocksdb_disable_wal`               | false                      |  禁用 rockdb 的 WAL
```

Comment:
alignment

Reply:
Have checked the review, the alignment is good, thank for your remind.